target/consensource-ui
# Add tag events to track page views

## Proposed change/fix

- Set the GA ID to the window to access it from subpages
- Add pageview event to each app's routing onmatch handler
- Format index files
- Add gtag and GA ID to be globals in eslint

Note: Most changes are formating. To see example of the actual substantial change, [here](https://github.com/target/consensource-ui/pull/27/files#diff-b7ad11343341c2aa9d0512f891c4a034L104) is an example adding the page view event generation to the routing.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

Rebuild the ui and run the project. Visit any persona page and open up the browser console. Type `dataLayer` to view the current information sent to GA. Then click on any links on the page, and log `dataLayer` again. Look at the last value in this list, and ensure `page_path` corresponds to the url after the anchor in the browser.
